.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -167,7 +166,6 @@ jobs:
           TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
           TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
       - name: Post success to Slack channel dev-connectors-extensibility
-        if: success()
```

Comment:
I think we still want to keep this one - We don't want to post success to slack every case, only if we were successful (Different from the one you removed in `Commit and push changes` , which we want it to attempt to do each time)